### PR TITLE
Update codeblocks project to support OpenAL soft extensions again

### PR DESCRIPTION
**Pull request by Lwmte**
Also remove super-old integrated AL sources from project and fix config path

NOTE: I didn't fix OpenAL paths, cause seems it works as is for everyone except me

### CODE REVIEW

**Comment by vvs-** [comment]
Seems that it doesn't compile.

File: src/audio.cpp [hunk 1]
```diff
@@ -4,13 +4,14 @@
 
 #include <math.h>
 
-#include "config-opentomb.h"
+#include "config-default/config-opentomb.h"
```
Comment (TeslaRus) [inline src/audio.cpp]: may be it is a reason of Travis failed...

Comment (vvs-) [inline src/audio.cpp]: Yes, that's wrong. There are platform dependent includes which won't work on another platform.

**Comment by TeslaRus** [comment]
here (https://github.com/TeslaRus/OpenTomb) AL reverb is enabled (C::B project is actual) and sources may be updated from main repo just by copy - replace all - all works

File: src/audio.cpp [hunk 2]
```diff
@@ -11,6 +11,7 @@ extern "C" {
 #include <alc.h>
 #ifdef HAVE_ALEXT_H
 #include <alext.h>
+#define AL_ALEXT_PROTOTYPES
```
Comment (TeslaRus) [inline src/audio.cpp]: should be in **config-opentomb.h**
after changes may be merged

Comment (Lwmte) [inline src/audio.cpp]: done ;)

**Comment by TeslaRus** [comment]
lgtm

**Pull request merged**